app/src/main/java/com/example/jasmiensofiecels/wordly/service/repository/DictionaryRepository.java

```diff
@@ -0,0 +1,77 @@
+package com.example.jasmiensofiecels.wordly.service.repository;
+
+import android.arch.lifecycle.MutableLiveData;
+import android.util.Log;
+
+import com.example.jasmiensofiecels.wordly.service.model.OxfordEntry.Example;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * The purpose of this class is to call the Oxford Dictionary API and
+ * return a Gson Example object to the observable's subscriber.
+ *
+ * Created by Jasmien Cels on 03/04/2018.
+ */
+
+@Singleton
+public class DictionaryRepository {
+
+    @Inject
+    public DictionaryRepository() {}
+
+    private final String HTTPS_OXFORD_URL = "https://od-api.oxforddictionaries.com:443/api/v1/";
+
+    public MutableLiveData<Example> getWordOfTheDay(String sourceLang,  MutableLiveData<String> wordID) {
+
+        final MutableLiveData<Example> results = new MutableLiveData<>();
+
+        //Serialize to Gson
+        Gson gson = new GsonBuilder().setLenient().create();
+
+        //Set up Retrofit call
+        Retrofit retrofit = new Retrofit.Builder()
```

Comment:
You should never call `new` in non-UI classes (except for Dagger classes). This will make unit testing really difficult. I know you haven't unit tested this class yet but try and do that next. You won't be able to mock responses for this as you are creating a new instance that you don't have access to in the test